src/core/server/stacks/helpers/retrieveParent.ts

```diff
@@ -31,13 +30,6 @@ async function retrieveParent(
     throw new CommentNotFoundError(input.parentID);
   }
 
-  // Check to see that the most recent revision matches the one we just replied
-  // to.
-  const revision = getLatestRevision(parent);
-  if (revision.id !== input.parentRevisionID) {
-    throw new CommentRevisionNotFoundError(parent.id, input.parentRevisionID);
-  }
-
```

Comment:
alright, now I'm tad bit confused, we were always checking against latest revision this whole time?

Does the check added to `createComment` matter then? Or was this only being called for `retrieveParent` which wasn't being used in `createComment`?

Hmmm... I'm going to have to test this to see what's going on here.

Reply:
Updating with what we discussed offline; removing this check altogether since we don't think it's necessary to check if replies are being made to stale comments or if replies to stale comment versions are being edited.